discovery-provider/src/queries/get_top_users.py

```diff
@@ -1,23 +1,32 @@
-from sqlalchemy.sql.expression import desc
-from src.models.models import AggregateUser, User
-from src.queries.query_helpers import helpers, paginate_query, populate_user_metadata
+from src.queries.query_helpers import get_pagination_vars, populate_user_metadata
 from src.utils.db_session import get_db_read_replica
 
+sql = """
+select users.*
+from users
+join aggregate_user using (user_id)
+where
+    is_current
+    and user_id in (
+        select user_id from aggregate_user
+        where track_count > 0
+        order by follower_count desc
+        limit :limit
+        offset :offset
+    )
+order by follower_count desc, user_id asc;
+"""
+
 
 def get_top_users(current_user_id):
```

Comment:
Probably worth adding a test for get_top_users as well, similar to https://github.com/AudiusProject/audius-protocol/blob/master/discovery-provider/integration_tests/queries/test_populate_track_metadata.py#L17

these tests can be run with docker running & ./scripts/test.sh to set up the initial harness, and then pytest <file> to test